spec/active_record/inheritance_spec.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveRecord
+  describe 'Base' do
```

Comment:
Ah, please do use `RSpec.describe`: that allows RSpec to run in its "no monkey-patch" mode, much less internal magic.